__tests__/components/ErrorTooltip.spec.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * (C) Symbol Contributors 2022
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and limitations under the License.
+ *
+ */
+import ErrorTooltip from '@/components/ErrorTooltip/ErrorTooltip.vue';
+import { ErrorTooltipTs } from '@/components/ErrorTooltip/ErrorTooltipTs';
+import { getComponent } from '@MOCKS/Components';
+
+describe('components/ErrorTooltip', () => {
+    const getErrorTooltipWrapper = (errors?: string[]) => {
+        return getComponent(
+            ErrorTooltip,
+            {},
+            {},
+            {
+                errors,
+            },
+            {},
+        );
+    };
+
+    test('renders error tooltip', () => {
+        // Arrange:
+        const wrapper = getErrorTooltipWrapper();
+
+        // Act + Assert:
+        expect(wrapper.find('tooltip').exists()).toBe(true);
+    });
+
+    test('field error returns null when no errors', () => {
+        // Arrange:
+        const wrapper = getErrorTooltipWrapper();
+        const component = wrapper.vm as ErrorTooltipTs;
+
+        // Assert:
+        expect(component.fieldError).toBe(null);
+        expect(component.displayedError).toBe('');
+        expect(component.errored).toBe(false);
+    });
+
+    test('field error returns null when errors are empty', () => {
+        // Arrange:
+        const errors = [];
+        const wrapper = getErrorTooltipWrapper(errors);
+        const component = wrapper.vm as ErrorTooltipTs;
+
+        // Assert:
+        expect(component.fieldError).toBe(null);
+    });
+
+    test('field error returns first error from error list', () => {
+        // Arrange:
+        const firstErrorItem = 'error 1';
+        const errors = [firstErrorItem, 'error 2'];
+        const wrapper = getErrorTooltipWrapper(errors);
+        const component = wrapper.vm as ErrorTooltipTs;
+
+        // Assert:
+        expect(component.fieldError).toBe(firstErrorItem);
+        expect(component.displayedError).toBe(firstErrorItem);
+        expect(component.errored).toBe(true);
+    });
+});
```

Comment:
consider adding test where fieldError != displayedError

Reply:
can you elaborate on this a little bit more pls? they are expected to be the same.
L70 and L71 =>
https://github.com/symbol/desktop-wallet/blob/5f3f6a485191b76f9b68b35ecdd0ab41fecd0293/__tests__/components/ErrorTooltip.spec.ts#L71

Reply:
i meant add test that changes to value to 'something' and then to '' or undefined.

Reply:
Refactored the component so that it can show multiple errors as a concatenated string.
 For given errors:['Error 1', 'Error 2', '', ' '] it will filter out the empty strings and present the following:
* Error 1
* Error 2